js/components/discussions/thread.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+<div>
+    <div class="list-group-item" id="{{ discussionIdAttr }}" @click="toggleDiscussions">
+        <div class="format-label pull-left">
+            <avatar :user="question.posted_by"></avatar>
+        </div>
+        <span class="list-group-item-link">
+            <a href="#{{ discussionIdAttr }}"><span class="fa fa-link"></span></a>
+        </span>
+        <h4 class="list-group-item-heading ellipsis open-discussion-thread">
+            <span>{{ discussion.title }}</span>
+        </h4>
+        <p class="list-group-item-text ellipsis open-discussion-thread list-group-message-number-{{ discussion.id }}">
+            {{ _('Discussion started on') }} {{ discussion.created | dt }} {{ _('with') }}
+            {{ _('nbmessages', {nb: responses.length}) }}
+        </p>
+    </div>
+    <div v-for="(index, response) in responses" id="{{ discussionIdAttr }}-{{ index }}"
+        class="list-group-item list-group-indent animated discussion-messages-list"
+        :class="{'body-only': index == 0, 'hidden': !detailed}">
+        <template v-if="index > 0">
+            <div class="format-label pull-left">
+                {{ response.posted_by | display }}
+            </div>
+            <span class="list-group-item-link">
+                <a href="#{{ discussionIdAttr }}-{{ index }}"><span class="fa fa-link"></span></a>
+            </span>
+        </template>
+        <p class="list-group-item-heading">
+            {{ response.content | markdown }}
+        </p>
+    </div>
+    <a v-if="!discussion.closed"
+        class="list-group-item add new-comment list-group-indent animated"
+        :class="{hidden: formDisplayed || !detailed}"
+        @click="displayForm">
+        <div class="format-label pull-left">+</div>
+        <h4 class="list-group-item-heading">
+            {{ _('Add a comment') }}
+        </h4>
+    </a>
+    <div class="list-group-item list-group-form list-group-indent animated"
+        id="{{ discussionIdAttr }}-{{ position }}"
```

Comment:
I think `position` declaration is missing somewhere (either `props`, `data` or `computed`)